### ISSUE TITLE
Using `...` syntax on exposed API.

### ISSUE THREAD

**Issue by phenomnomnominal** [GoogleChromeLabs/comlink#505]
Hey team, I just had some fun debugging this! I have some code where I essentially do:

```
function blah (...args) {
   const api = Comlink.wrap(worker);
   return api[name](...args);
}
```

Where the splat function call transpiles to:

```
return api[name].apply(api, args);
```

The `.apply` triggers the `get` trap on the Proxy which returns another Proxy, and chaos ensued 😅

I guess you could maybe add another guard to the `get` trap for this case? For now I'll just target ES2015!

**Comment by surma** [comment]
(Side note: You really shouldn't be transpiling `...` anymore. Targeting ES2017 should be your default unless you have good reasons to do otherwise. Can only recommend the [this talk](https://www.youtube.com/watch?v=cLxNdLK--yI&list=PLNYkxOF6rcIDzLmWaDwfHVZJl1Q5RFgOR&index=6) by @housseindjirdeh and @developit.)

You are right, I do have [a handler for `bind()`](https://github.com/GoogleChromeLabs/comlink/blob/520e4d64072bc07c3c60dd67dd95910a23602445/src/comlink.ts#L430-L432), but not for `apply()`. I should add one. Or do you wanna whip up a PR? :D

**Comment by phenomnomnominal** [comment]
Just saw that talk the other day 😎 , I'll admit I'm still a little iffy about taking the leap! 

But yeah I'll have a crack at a PR, just wanted to validate that it's an issue first! I imagine it is possible this could come up in non-transpiled code 😊

**Comment by surma** [comment]
Yeah, definitely a good ideal to add something for `apply()` either way (maybe also `call()` :thinking:). Looking forward to your PR! Make sure to add a test so we don't regress in the future.

**Comment by negue** [comment]
@surma could we merge this PR for that? :)

**Comment by Aaronius** [comment]
It's worth considering that each method that's handles specially within the proxy, be it `bind`, `apply`, `call`, `then`, or otherwise, is another method the consumer can't expose from the remote if they were to want to. In other words, if the remote tried to expose an object with a legitimate method called `apply` on it, we wouldn't be able to call it from our local proxy.
